### ISSUE TITLE
PHP 8.1 Deprecation messages

### ISSUE THREAD

**Issue by philsturgeon** [infobip/infobip-api-php-client#47]
```
PHP Deprecated:  Return type of Infobip\Model\SmsInboundMessageResult::offsetExists($offset) should either be compatible with ArrayAccess::offsetExists(mixed $offset): bool, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in /Users/phil/src/infobip/vendor/infobip/infobip-api-php-client/Infobip/Model/SmsInboundMessageResult.php on line 296

Deprecated: Return type of Infobip\Model\SmsInboundMessageResult::offsetExists($offset) should either be compatible with ArrayAccess::offsetExists(mixed $offset): bool, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in /Users/phil/src/infobip/vendor/infobip/infobip-api-php-client/Infobip/Model/SmsInboundMessageResult.php on line 296
PHP Deprecated:  Return type of Infobip\Model\SmsInboundMessageResult::offsetGet($offset) should either be compatible with ArrayAccess::offsetGet(mixed $offset): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in /Users/phil/src/infobip/vendor/infobip/infobip-api-php-client/Infobip/Model/SmsInboundMessageResult.php on line 308

Deprecated: Return type of Infobip\Model\SmsInboundMessageResult::offsetGet($offset) should either be compatible with ArrayAccess::offsetGet(mixed $offset): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in /Users/phil/src/infobip/vendor/infobip/infobip-api-php-client/Infobip/Model/SmsInboundMessageResult.php on line 308
PHP Deprecated:  Return type of Infobip\Model\SmsInboundMessageResult::offsetSet($offset, $value) should either be compatible with ArrayAccess::offsetSet(mixed $offset, mixed $value): void, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in /Users/phil/src/infobip/vendor/infobip/infobip-api-php-client/Infobip/Model/SmsInboundMessageResult.php on line 321

Deprecated: Return type of Infobip\Model\SmsInboundMessageResult::offsetSet($offset, $value) should either be compatible with ArrayAccess::offsetSet(mixed $offset, mixed $value): void, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in /Users/phil/src/infobip/vendor/infobip/infobip-api-php-client/Infobip/Model/SmsInboundMessageResult.php on line 321
PHP Deprecated:  Return type of Infobip\Model\SmsInboundMessageResult::offsetUnset($offset) should either be compatible with ArrayAccess::offsetUnset(mixed $offset): void, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in /Users/phil/src/infobip/vendor/infobip/infobip-api-php-client/Infobip/Model/SmsInboundMessageResult.php on line 337

Deprecated: Return type of Infobip\Model\SmsInboundMessageResult::offsetUnset($offset) should either be compatible with ArrayAccess::offsetUnset(mixed $offset): void, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in /Users/phil/src/infobip/vendor/infobip/infobip-api-php-client/Infobip/Model/SmsInboundMessageResult.php on line 337
PHP Deprecated:  Return type of Infobip\Model\SmsInboundMessageResult::jsonSerialize() should either be compatible with JsonSerializable::jsonSerialize(): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in /Users/phil/src/infobip/vendor/infobip/infobip-api-php-client/Infobip/Model/SmsInboundMessageResult.php on line 349

Deprecated: Return type of Infobip\Model\SmsInboundMessageResult::jsonSerialize() should either be compatible with JsonSerializable::jsonSerialize(): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in /Users/phil/src/infobip/vendor/infobip/infobip-api-php-client/Infobip/Model/SmsInboundMessageResult.php on line 349
```

**Comment by felipeteko** [comment]
+1

**Comment by veugenio** [comment]
+1

**Comment by ib-fsrnec** [comment]
Hi @philsturgeon @felipeteko @veugenio, thank you very much for reaching us and reporting the issue. We expect this to be covered in our next release that should be out there in a few weeks from now. Please follow the thread and release notes for our future updates.

**Comment by ibirogar** [comment]
FIxed in [4.0.0](https://github.com/infobip/infobip-api-php-client/releases/tag/4.0.0)
